### ISSUE TITLE
Take Over Mode not working properly - TS Errors without Take Over

### ISSUE THREAD

**Issue by ragokan** [johnsoncodehk/volar#1169]
1- Most of quick fix options does not work on *TS* files.
2- Suggestions does not show right props.

**This video is recorded on initial project that is created with `npm init vue@latest`**
https://user-images.githubusercontent.com/63511519/162743102-70c8e8f6-4ab5-45ba-a241-c28842efc391.mov

-----
When I disable Take Over mode, even in the initial beginning project, I get these errors;
![image](https://user-images.githubusercontent.com/63511519/162743423-ece3442b-b0e4-42f0-98a2-a8c0d28d558a.png)
Basically Vue TS does not recognize the *.Vue* files.

**Comment by yaegassy** [comment]
@ragokan For projects created with `create-vue`, Try adding `declare module "*.vue" ...` to `env.d.ts`.

**Example**:

```typescript
/// <reference types="vite/client" />

declare module "*.vue" {
  import type { DefineComponent } from "vue";
  // eslint-disable-next-line @typescript-eslint/no-explicit-any, @typescript-eslint/ban-types
  const component: DefineComponent<{}, {}, any>;
  export default component;
}
```

**Comment by yaegassy** [comment]
Do you have `node_modules` in your project? try, `npm i` or `pnpm i` or `yarn`

**Comment by ragokan** [comment]
> Do you have `node_modules` in your project? try, `npm i` or `pnpm i` or `yarn`

Yes, I do. It is not my first project, too.

**Comment by yaegassy** [comment]
Your .mov appears to have no `node_modules` and no lock file.

<img width="642" alt="volar-issue-1169-1" src="https://user-images.githubusercontent.com/188642/162751248-0417fe65-92f1-4a0b-8e6b-fe3365a7dc94.png">

**Comment by ragokan** [comment]
> Your .mov appears to have no `node_modules` and no lock file.
> 
> 
> 
> <img width="642" alt="volar-issue-1169-1" src="https://user-images.githubusercontent.com/188642/162751248-0417fe65-92f1-4a0b-8e6b-fe3365a7dc94.png">
> 
> 
> 
> 

They are just hidden in vscode

**Comment by yaegassy** [comment]
See here. Have you disabled `@builtin typescript`? <https://github.com/johnsoncodehk/volar/discussions/471#discussioncomment-1361669>

**Comment by yaegassy** [comment]
I can't reproduce it in my environment. The only other thing I can think of is to restart VSCode. I can't reproduce it, so I have no further advice... :(

**Comment by yaegassy** [comment]
Perhaps `github.dev`? There may be a problem with the browser extension. In that case, try the standalone VSCode app.

**Comment by extrem7** [comment]
Take over mode also became broken in all my projects after the weekend
![image](https://user-images.githubusercontent.com/24792876/162793497-f1ea9561-1412-46a4-88ed-c2abf9dc6219.png)

**Comment by ragokan** [comment]
> Take over mode also became broken in all my projects after the weekend ![image](https://user-images.githubusercontent.com/24792876/162793497-f1ea9561-1412-46a4-88ed-c2abf9dc6219.png)

Yeah, in my previous projects, I didn't have any errors, I just created this one yesterday, but errors started after the updates in extension.
